plugins/utils/todo-comments.nix

```diff
@@ -76,8 +76,8 @@ in
               TODO = { icon = " "; color = "info"; };
               HACK = { icon = " "; color = "warning"; };
               WARN = { icon = " "; color = "warning"; alt = [ "WARNING" "XXX" ]; };
-              PERF = { icon = " "; alt = [ "OPTIM" "PERFORMANCE" "OPTIMIZE" ]; };
-              NOTE = { icon = " "; color = "hint"; alt = [ "INFO" ]; };
```

Comment:
Same as other, looks like a different default icon:
`NOTE = { icon = " ", color = "hint", alt = { "INFO" } },`

Reply:
They should still be nix attributes. I was just posting the lua from the plugin.